docs/language/functions/regexp_replace.md

```diff
@@ -10,28 +10,50 @@ regexp_replace(s: string, re: string|regexp, new: string) -> string
 ### Description
 
 The _regexp_replace_ function substitutes all characters matching the
-[regular expression](../overview.md#regular-expressions) `re` in string `s` with
+[regular expression](../overview.md#811-regular-expressions) `re` in string `s` with
 the string `new`.
 
 Variables in `new` are replaced with corresponding matches drawn from `s`.
-A variable is a substring of the form $name or ${name}, where name is a non-empty
-sequence of letters, digits, and underscores. A purely numeric name like $1 refers
-to the submatch with the corresponding index; other names refer to capturing
-parentheses named with the `(?P<name>...)` syntax. A reference to an out of range or
+A variable is a substring of the form `$name` or `${name}`, where `name` is a non-empty
+sequence of letters, digits, and underscores. A purely numeric name like `$1` refers
+to the submatch with the corresponding index; other names refer to parenthesized
+capturing groups named with the `(?P<name>...)` syntax. A reference to an out of range or
```

Comment:
In the top of this doc there's a link to the [regular expressions section](https://zed.brimdata.io/docs/language/overview#811-regular-expressions) of the Zed language overview doc, which in turn points to the [RE2](https://github.com/google/re2) repo, so strictly speaking I guess it's fair game to quite the RE2 docs verbatim like we'd done here. However, it still seems likely users may try to work off this doc in isolation, so I wanted to improve the wording a bit to help along those that don't have as much regexp experience. While "capturing parentheses" is indeed a phrase used in the [Expand docs](https://pkg.go.dev/regexp#Regexp.Expand) being quoted here, the phrase "parenthesized subexpressions (also known as capturing groups)" appears higher up in that same doc, so I've shortened that and proposed it instead since I happen to find it clearer.

Reply:
I like the original "capturing parentheses" better.

Reply:
@nwt: Ok, I've put back "capturing parentheses".